enabler/src/main/java/com/openxc/enabler/OpenXcEnablerActivity.java

```diff
@@ -159,7 +163,11 @@ static String getBugsnagToken(Context context) {
     @Override
     protected void onResume() {
         super.onResume();
-        checkForCrashes();
+        try {
+            checkForCrashes();
+        }catch (NoClassDefFoundError e){
+            Log.w(TAG, "Busgnag is unsupported when building from Eclipse", e);
```

Comment:
could use a better error message like "Failed checkForChrashes call"
also could dump the stack with a line
"e.printStackTrace();"